static/css/login.scss

```diff
@@ -1,30 +1,134 @@
-@import "variables";
-
-body {
-  display: flex;
-  justify-content: center;
-  background-color: $lighter-grey;
-
-  .login-container {
-    // background-color: $lighter-grey;
-    width: 700px;
-    margin-top: 100px;
-
-    form {
-      display: flex;
-      flex-direction: column; 
-      
-      input {
-        margin-top: 20px;
-        padding: 8px;
-        border-radius: 6px;
-        border: 1px black solid;
-
-        &:focus {
-          outline: none;
-          border: 2px solid $blue;
-        }
-      }
-    }
-  }
+@import url(https://fonts.googleapis.com/css?family=Exo:100,200,400);
+@import url(https://fonts.googleapis.com/css?family=Source+Sans+Pro:700,400,300);
+@import url('https://fonts.googleapis.com/css2?family=Chango&display=swap');
+
+body{
+	margin: 0;
+	padding: 0;
+	background: #fff;
+
+	color: #fff;
+	font-family: Arial;
+	font-size: 12px;
+}
+
+.body{
```

Comment:
It is very uncommon to style the body element absolute with negative placement. could you check if there is a better option?